lib/Arch/AArch32/Decode.cpp

```diff
@@ -1120,10 +1127,31 @@ static bool EvalPCDest(Instruction &inst, const bool s, const unsigned int rd,
       inst.category = Instruction::kCategoryError;
       return false;
     } else {
-      auto src1 = EvalOperand(inst, inst.operands[1]);
-      auto src2 = EvalOperand(inst, inst.operands[2]);
 
-      if (!src1 || !src2) {
+      // check if the number of operands in instruction is 5
```

Comment:
Explain this check in a comment, it's not obvious why `5` is significant or always right for all users of `EvalPCDest`.

Reply:
Updated comments about the operand count check. This is a hack and I will relook the implementation of `EvalPCDest`. The previous implementation was not doing anything because it was not evaluating the source operands and also not handling the expression operands.